Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -181,8 +181,8 @@ public struct Configuration: Equatable {
     public func lintablePathsForPath(path: String,
                                      fileManager: NSFileManager = fileManager) -> [String] {
         let pathsForPath = included.isEmpty ? fileManager.filesToLintAtPath(path) : []
-        let excludedPaths = excluded.flatMap(fileManager.filesToLintAtPath)
-        let includedPaths = included.flatMap(fileManager.filesToLintAtPath)
+        let excludedPaths = excluded.flatMap{fileManager.filesToLintAtPath($0, rootDirectory: self.rootPath)}
```

Comment:
Please omit `self.` since it can be inferred, as per the Swift style guide we follow: https://github.com/github/swift-style-guide#only-explicitly-refer-to-self-when-required

Also, a single space after `{` and before `}`:

```swift
flatMap{ fileManager.filesToLintAtPath($0, rootDirectory: rootPath) }
```